instrumentation/spring/spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -132,6 +132,7 @@ testing {
         implementation("org.springframework.boot:spring-boot-autoconfigure:$springBootVersion")
 
         implementation(project(":instrumentation:logback:logback-appender-1.0:library"))
+        compileOnly(project(":instrumentation:logback:logback-mdc-1.0:library"))
```

Comment:
I didn't follow how this `compileOnly` dep is used?

Reply:
I changed the scope to `implementation`, hopefully that is less confusing. A class from the `logback-mdc-1.0:library` is used in a test so it is needed to compile the test, I think it gets added to runtime dependencies because the main project depends on it and the suite depends on the main project.